Gemfile

```diff
@@ -44,6 +44,7 @@ gem 'config'
 gem 'ckeditor'
 gem 'rails-i18n'
 gem 'devise-i18n'
+gem 'will_paginate'
```

Comment:
như này thì lại confict ò :))

Reply:
ta chờ PR của mi đc merge rồi ta fix =))
phải cho vào để preview xem nó chạy ko :v

Reply:
a có thể rebase nhánh của Toàn trước cũng đc. xong về sau Toàn đc merge trc là ok